examples/hybrid/driver.jl

```diff
@@ -88,12 +88,14 @@ else
         h_space = make_horizontal_space(horizontal_mesh, npoly)
         comms_ctx = nothing
     end
-    center_space, face_space = make_hybrid_spaces(h_space, z_max, z_elem)
+    z_stretch_scale = FT(7e3)
+    center_space, face_space =
+        make_hybrid_spaces(h_space, z_max, z_elem; z_stretch_scale)
     ᶜlocal_geometry = Fields.local_geometry_field(center_space)
     ᶠlocal_geometry = Fields.local_geometry_field(face_space)
     Y = Fields.FieldVector(
-        c = center_initial_condition.(ᶜlocal_geometry),
-        f = face_initial_condition.(ᶠlocal_geometry),
+        c = center_initial_condition(ᶜlocal_geometry),
+        f = face_initial_condition(ᶠlocal_geometry),
```

Comment:
@dennisYatunin my question was on the returned FieldVector here that now conflicts with the use in `default_cache`